Python/marshal.c

```diff
@@ -1364,6 +1375,11 @@ r_object(RFILE *p)
             PyObject* linetable = NULL;
             PyObject *exceptiontable = NULL;
 
+            if (!p->allow_code) {
+                PyErr_SetString(PyExc_ValueError,
+                                "loading of code objects is prohibited");
```

Comment:
I suggest "disallowed" to be closer to "allow":

```suggestion
                                "loading of code objects is disallowed");
```